js/binance.js

```diff
@@ -67,8 +67,7 @@ module.exports = class binance extends Exchange {
                 'www': 'https://www.binance.com',
                 'referral': 'https://www.binance.com/?ref=10205187',
                 'doc': [
-                    'https://github.com/binance-exchange/binance-official-api-docs/blob/master/rest-api.md',
-                    'https://github.com/binance-exchange/binance-official-api-docs/blob/master/wapi-api.md',
+                    'https://binance-docs.github.io/apidocs/spot/en',
```

Comment:
Hey! ) Are you sure that this is the official doc?

Reply:
Pardon, looks like they are not pointing to this repo from their website, so, yes, this is the correct URL. Thx for the update!